bcipy/signal/evaluate/evaluate.py

```diff
@@ -0,0 +1,49 @@
+
+"""evaluate.py """
+
+
+from bcipy.signal.evaluate.rules import *
```

Comment:
never star import

Reply:
amended! is now: from rules import HighVoltage, LowVoltage